content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -174,7 +172,7 @@ The following resources are created as part of the profile creation:
   - A Kubernetes namespace that shares the same name with the corresponding
     profile.
   - Kubernetes RBAC role binding for the namespace: *Admin*. This makes the
```

Comment:
Suggestion:

- a11y: To help new users, maybe we can spell out certain abbreviations/terminology, even if it's common knowledge (Kubernetes has so many interesting terms, it needs its own dictionary 🙂)

```diff
-  - Kubernetes RBAC role binding for the namespace: *Admin*. This makes the
+ Kubernetes RBAC ([Role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac/))
+ role binding for the namespace: *Admin*. This makes the
...
```

Reply:
Good point!